src/index.ts

```diff
@@ -16,15 +16,18 @@ import {
   getProjectDirPathInOutDir,
   loadConfig
 } from './helpers';
-import { Output } from './utils';
+import { Output, resolveFullImportPaths } from './utils';
+
+export interface ReplaceTscAliasPathsOptions {
```

Comment:
Moved the options into an exported interface to make it accessible to API consumers.